src/empty.js

```diff
@@ -3,7 +3,7 @@ var _hasMethod = require('./internal/_hasMethod');
 
 
 /**
- * `empty` wraps any object in an array. This implementation is compatible with the
+ * `empty` returns an empty list for any argument. This implementation is compatible with the
```

Comment:
looking over the docs, we are documenting the list case, and leaving the dispatch case as a delightful easter egg for users to discover.